src/main/java/frc/robot/commands/CoralDepositorCommand.java

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.CoralDepositorControl;
+import frc.robot.subsystems.IntakeSubsystem;
+
+public class CoralDepositorCommand extends Command {
+    private CoralDepositorControl coralDepositorControl;
+    private boolean direction;
+    private boolean bUseSensor;
+  /** Creates a new IntakeControl. */
+
+  /**
+   * @brief
+   * @param feedNote true to feed it, false to back it 
+   * @param bUseSensor true with sensor
+      * @return 
+      * @return
+      */
+    public CoralDepositorCommand(boolean feedNote) {
+      this.direction = feedNote;
+      coralDepositorControl = CoralDepositorControl.getInstance();
+
+    // Use addRequirements() here to declare subsystem dependencies.
+    if (coralDepositorControl != null) {
+      addRequirements(coralDepositorControl);
```

Comment:
coralDepositorControl here is null.
You cancoralDepositorControl= CoralDepositorControl.getInstance();